gtfu/src/main/java/gtfu/tools/BlockDataGenerator.java

```diff
@@ -121,8 +124,9 @@ public BlockDataGenerator(String cacheFolder, String outputFolder, String agency
     }
 
     private static void usage() {
-        System.err.println("usage: BlockDataGenerator -a|--agency-id <agency-id> -u|--upload [-c|--cache-folder <cache-folder>] [-o|--output-folder <output-folder>] [-d|--date <mm/dd/yy>|<n>]");
+        System.err.println("usage: BlockDataGenerator -a|--agency-id <agency-id> [-U|--url <static-gtfs-url>] -u|--upload [-c|--cache-folder <cache-folder>] [-o|--output-folder <output-folder>] [-d|--date <mm/dd/yy>|<n>]");
         System.err.println("    <agency-id> a transit agency identifier constructed from the alphabet of [a-z\\-]");
+        System.err.println("    <static-gtfs-url> link to agency's static static GTFS data. System attempts to backfill with data from agency.yaml if omitted");
```

Comment:
How about:
Optional link to agency's static static GTFS data. For agencies present in agencies.yml, the URL will be loaded automatically